core/src/main/scala/org/apache/spark/sql/rapids/tool/ToolUtils.scala

```diff
@@ -317,6 +317,20 @@ object SQLMetricsStats {
   }
 }
 
+object IgnoreExecs {
+  val AdaptiveSparkPlan = "AdaptiveSparkPlan" // AdaptiveSparkPlan is not a real exec. It is
+  // a wrapper for the whole plan.
+  val CollectLimit = "CollectLimit" // Collect Limit replacement can be slower on the
+  // GPU. Disabled by default.
+  val ScanExistingRDD = "Scan ExistingRDD"
+  val ExecuteCreateViewCommand = "Execute CreateViewCommand"
+  val ExistingRDD = "ExistingRDD"
+  val LocalTableScan = "LocalTableScan"
+
+  def getallIgnoreExecs = Set(AdaptiveSparkPlan, CollectLimit, ScanExistingRDD,
```

Comment:
nit: Can we replace by `getAllIgnoreExecs` and add type annotation since it is a function definition?